src/index.html

```diff
@@ -11,7 +11,223 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
```

Comment:
no need blank line between parent and child tags
```suggestion
    <form
```